### DISCUSSION TITLE
Supported versions of Visual Studio Code

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [JosefPihrt/Roslynator#779] [Q&A] [answered]
Hello! What versions of Visual Studio Code are supported?

**Comment by josefpihrt** [comment, marked as answer]
More than VS Code version it is about version of [C# for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) extension.

Minimal required version of this extension is 1.21.13 (as mentioned [here](https://marketplace.visualstudio.com/items?itemName=josefpihrt-vscode.roslynator)).

Roslynator was updated recently to use Roslyn 3.8.0 which means that next release of Roslynator for VS Code will require C# for VS Code [1.37.4](https://github.com/OmniSharp/omnisharp-roslyn/blob/master/CHANGELOG.md#1374---2020-11-20) or higher.
